free-1.md

```diff
@@ -4,40 +4,120 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+A CAOS (Computer Aided Organic Synthesis) DSL.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Currently, chemists have one of three ways in which they can predict how a 
+reaction will occur, given certain molecules and conditions.
+
+1. Map it out by hand
+2. Look it up
+3. Use expensive, proprietary software
+
+For common, straightforward reactions, the first two are relatively easy to do.
+Unfortunately (actually this is a good thing for humanity, but lots of work
+for chemists) many professional chemists and researchers are interested in 
+new, difficult reactions that haven't been seen before.  Thus they either have
+to use expensive software that eats up their grant money (and may not meet 
+their needs) or they have to take a long time to work it out by hand.  With a
+free tool (language) designed to be both expressive and intuitive, this process
+could be greatly improved (especially if the tool/language were designed to work
+alongside existing databases of reaction pathways).
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A reaction is generally predicted using well-specified rules that apply to 
+specific molecules and conditions.  This lends itself well to an imperative 
+style such as
+
+```
+Molecule hydronium = OH-
+Molecule hydroxide = H30+
+Molecule[] products = react(hydronium, hydroxide, conditions)
+products == [(Molecule) H20, (Molecule) H20]
+```
+
+ALternatively, when seeking synthesis pathways (i.e. a known endpoint) recursion 
+works well, such as
+
+```
+Molecule endproduct = <something complicated>
+Molecule startingpoint = synthesize(endproduct)
+```
+
+where `synthesize` is some sort of recursive function.
+
+It meshes well with existing programming language paradigms and designs.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+What excites me about this is that I've been working on something similar since 
+my sophomore year, when I was still a premed student taking Organic Chemistry.
+
+I thought to myself that these rules were relatively straightforward, and it was
+very much an algorithmic process.  I've been working on it off and on since 
+then.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Predicting organic chemistry reaction pathways
```

Comment:
Because I know a lot more about organic chemistry.  It could probably be extended to inorganic chemistry, but I have even less domain knowledge there.